spark/src/main/resources/R/zeppelin_sparkr.R

```diff
@@ -31,7 +31,7 @@ print(paste("LibPath ", libPath))
 library(SparkR)
 
 
-SparkR:::connectBackend("localhost", port, 6000)
+SparkR:::connectBackend("localhost", port)
```

Comment:
Can you check the original jira issue? I'm not good at R and sparkR but it looks related.